### DISCUSSION TITLE
AQ40 Idol/Key drop rate.

### DISCUSSION THREAD

**Discussion by Zelfie** [chromiecraft/chromiecraft#4383] [Suggestions]
Is it currently Blizzlike? Even if it is please consider buffing it considerably.
We're a few weeks into AQ40 now and the droprates are exhaustingly bad.
We've cleared every boss, and the idols/keys have been so rare that most people with a tier token still can't hand them in.
The backlog of people with tokens is already getting a bit ridiculous in my guild and the problem will only seem to get worse. Actually obtaining a full T2.5 set feels like an impossible goal at the moment, especially with TBC feeling so close on the Horizon. With a full clear we're lucky if we get 4 idols, and there's 10 tier drops in a full raid?
Please consider fixing Blizzards awful bottleneck so we can actually enjoy the most difficult raid of Classic without sitting on tokens.

**Comment by sombritas** [comment]
it's indeed a problem both short term and long term, which as of now can't even be circumvented by trashfarming

**Comment by Peti321** [comment]
Even though we farm the trash it's not dropping. I don't think this should stay blizzlike if it is currently.

**Comment by balleny** [comment]
Sounds a bit like your group may not properly looting the mobs. The keys are common items and therefore not affected by loot master (if not using a script), and thus have to be looted individually.
We have seen usually more keys than coffers in the runs so far, and plenty of idols (10+).

Moreover, you can also consider farming the first trash packs before Skeram. Either everything up to Skeram followed by a reset, or killing only 3 out of 4 Sentinals and then running out of the dungeon for unlimited farm.

**Reply by Zelfie** [reply]
Cleared upto and including Twins, only 1 chest. So the second key I'm sitting on isn't much use.
If that's the solution, farming and resetting the first few mobs, then the droprate surely needs drastically increasing so that isn't the solution.

**Reply by balleny** [reply]
Just had another 8/9 run yesterday, and again, we had 8 keys drop and could open all 7 coffers (the number of coffers spawning is identical, just positions vary from ID to ID). Also 6 or 7 idols dropping from trash (and we didnt clear all the trash)
Seems totally blizzlike to me. Rng included, I would still advise you have your raid members check loot thoroughly.

**Comment by Brylezor** [comment]
considering the relative short time till tbc comes out when the gear becomes somewhat irrelevant, maybe they should reduce the quest requirements (if even possible) or up the droprate

**Reply by Brylezor** [reply]
also Balleny- its clear that there are a few people coming up with the same issue, you cant just dismiss all of us with the same "sounds a bit like your group may not properly looting...." 
multiple groups encountering the same issue

**Comment by shaszun** [comment]
We are getting like 4-6 idols per run without coffers. So far 2 keys for 3 weeks and 10 tokens per run require 20 idols. I think making the drop rate doubled/triple will be the best solution.

**Reply by balleny** [reply]
Just had another 8/9 run yesterday, and again, we had 8 keys drop and could open all 7 coffers. Also 6 or 7 idols dropping from trash (and we didnt clear all the trash)
Seems totally blizzlike to me. Rng included, I would still advise you have your raid members check loot thoroughly.

**Comment by AcetylGer** [comment]
We currently have a waiting list of 21 hand-ins that we are missing idols for and thats just after 2 weeks.
The droprate is way to low.
We are looting every dead mob for the rep items so no, we are not missing keys.

**Comment by Amrx2** [comment]
I'll just copy this over from the other suggestion regarding this so it isn't missed, will lock this one.

_Hi all, just to add that we have been taking a deep dive into all available data we have for these drop rates, including classic era servers, other server cores etc. What we have found is our drop rates on idols are already higher than all sources we can find. That would suggest a nerf would be in order, don't worry we aren't going to do that.

Whilst you are getting 2 months of AQ40 almost before TBC opens, please don't forget that a good chunk of AQ40 gear will be massively relevant in the 64 bracket (which you have until February 2023).

So as we currently already have the drops higher than they should be, we won't be making any further changes at this moment in time to these.

Cheers,
Construct_
